### DISCUSSION TITLE
Select element's label not floated on intial load

### DISCUSSION THREAD

**Discussion by robodna** [mdbootstrap/TW-Elements#2147] [Chat about dev stuff]
UPDATE: This issue has been resolved and was not related to the Select component.

I'm using the Select element with SolidJS.  I call iniTE({Select}) in my SolidJS component's onMount.  

I also have a component which uses the Select element wrapped in a SolidJS <Show> element which seems to be causing an issue.  On the first load, the Select's label is not floating above the input.  The value is correct.  If I click on the select box and then close it, the label floats up.

I have noticed this happens when I use attributes to init the Select element instead of Javascript.  I solved this issue with the Input element by using javascript to init Select in the onMount, but using Javascript to init the Select element in the same way does not fix the label issue.

Any ideas on how I can init the Select element so that its label is rendered correctly on first load?

Thank you!

**Comment by juujisai** [comment]
Hi! I'm glad that you have found an answer for your issue! Since the issue wasn't related to the TWE I am closing the discussion.

Best Regards!
